etc/data/viber_attacked_by_syrian_electronic_army.json

```diff
@@ -1,3 +1,3 @@
t-custom menu-item-899772\"><a href=\"http://techcrunch.cn/\">China</a></li>\n<li id=\"menu-item-899773\" class=\"menu-item menu-item-type-taxonomy menu-item-object-category menu-item-899773\"><a href=\"http://techcrunch.com/europe/\">Europe</a></li>\n<li id=\"menu-item-899774\" class=\"menu-item menu-item-type-custom menu-item-object-custom menu-item-899774\"><a href=\"http://jp.techcrunch.com/\">Japan</a></li>\n</ul>\n</div>\n\t\t\t\t</div>\n\t\t\t\t\n\n\t\t\t</div>\n\t\t</nav><div class=\"footer-follow g g-3up\">\n\n\t\t\t\n\t\t\t<div class=\"gi\">\n\t\t\t\t<div class=\"footer-social\">\n\t\t\t\t\t<h3>Follow TechCrunch</h3>\n\t\t\t\t\t<ul class=\"inline-list social-list sprite-social\">\n<li><a href=\"https://www.facebook.com/techcrunch\" rel=\"external\" class=\"spricon i-facebook-2\" target=\"_blank\"><span class=\"is-vishidden\">Facebook</span></a></li>\n\t\t\t\t\t\t<li><a href=\"https://twitter.com/techcrunch\" rel=\"external\" class=\"spricon i-twitter-2\" target=\"_blank\"><span class=\"is-vishidden\">Twitter</span></a></li>\n\t\t\t\t\t\t<li><a href=\"https://plus.google.com/+TechCrunch\" rel=\"external\" class=\"spricon i-google-plus-2\" target=\"_blank\"><span class=\"is-vishidden\">Google+</span></a></li>\n\t\t\t\t\t\t<li><a href=\"http://www.linkedin.com/company/techcrunch\" rel=\"external\" class=\"spricon i-linkedin-2\" target=\"_blank\"><span class=\"is-vishidden\">LinkedIn</span></a></li>\n\t\t\t\t\t\t<li><a href=\"http://www.youtube.com/user/techcrunch\" rel=\"external\" class=\"spricon i-youtube-2\" target=\"_blank\"><span class=\"is-vishidden\">Youtube</span></a></li>\n\t\t\t\t\t\t<li><a href=\"http://pinterest.com/techcrunch/\" rel=\"external\" class=\"spricon i-pinterest\" target=\"_blank\"><span class=\"is-vishidden\">Pinterest</span></a></li>\n\t\t\t\t\t\t<li><a href=\"http://techcrunch.tumblr.com/\" rel=\"external\" class=\"spricon i-tumblr\" target=\"_blank\"><span class=\"is-vishidden\">Tumblr</span></a></li>\n\t\t\t\t\t\t<li><a href=\"http://instagram.com/techcrunch\" rel=\"external\" class=\"spricon i-instagram\" target=\"_blank\"><span class=\"is-vishidden\">Instagram</span></a></li>\n\t\t\t\t\t\t<li><a href=\"http://www.stumbleupon.com/channel/techcrunch/\" rel=\"external\" class=\"spricon i-stumbleupon\" target=\"_blank\"><span class=\"is-vishidden\">StumbleUpon</span></a></li>\n\t\t\t\t\t\t<li><a href=\"/rssfeeds/\" rel=\"external\" class=\"spricon i-feed-2\"><span class=\"is-vishidden\">Feed</span></a></li>\n\t\t\t\t\t</ul>\n</div>\n\t\t\t</div>\n\t\t\t\n\n\t\t\t\n\t\t\t<div class=\"gi\">\n\t\t\t\t<h3>TechCrunch Apps</h3>\n\t\t\t\t<ul class=\"inline-list social-list sprite-social\">\n<li><a href=\"https://itunes.apple.com/us/app/techcrunch/id526058642?mt=8\" class=\"spricon i-apple\" rel=\"external\" target=\"_blank\"><span class=\"is-vishidden\">iOS</span></a></li>\n\t\t\t\t\t<li><a href=\"https://play.google.com/store/apps/details?id=com.aol.mobile.techcrunch\" class=\"spricon i-android\" rel=\"external\" targe
```

Comment:
What actually changed in this file? Upon visual inspection they look about identical.

Reply:
`<doctype` declaration was added.